### PR TITLE
[INLONG-9918][Audit] Audit-service add codes of config

### LINKED ISSUE #9918: [Feature][Audit] Audit-service add codes of config
### Description

Audit-service add codes of config

### Use case

Audit-service add codes of config

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by doleyzi**
### Prepare a Pull Request
- Fixes #9918
### Motivation

*Audit-service add codes of config*

### Modifications

*Audit-service add codes of config.*

### CODE REVIEW

File: inlong-audit/audit-service/src/main/java/config/ConfigConstants.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+package config;
+
+/**
+ * Config constants
+ */
+public class ConfigConstants {
+
+    // Source config
+    public static final String CLICKHOUSE_DRIVER = "clickhouse.driver";
```
Comment (vernedeng) [inline inlong-audit/audit-service/src/main/java/config/ConfigConstants.java:26]: KEY_CLICKHOUSE_DRIVER ?

File: inlong-audit/audit-service/src/main/java/source/SourceStat.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+/**
+ * Source stat interface.
+ */
+public interface SourceStat extends Runnable {
+
+    public void statByStep();
+
+    public void aggregate(String auditId);
+
+    public void doAggregate(String startTime, String endTime, String auditId);
```
[the remark this answers is not in the document]
Comment (doleyzi) [reply inlong-audit/audit-service/src/main/java/source/SourceStat.java]: yes, it has been changed to a better name

File: inlong-audit/audit-service/src/main/java/channel/DataQueue.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    /**
+     *  Push data
+     * @param statDataPo
+     */
+    public void push(StatData statDataPo) {
+        try {
+            queue.put(statDataPo);
+        } catch (InterruptedException e) {
```
Comment (aloyszhang) [inline inlong-audit/audit-service/src/main/java/channel/DataQueue.java:47]: why handle InterruptedException and just throw a RuntimeException?

File: inlong-audit/audit-service/src/main/java/channel/DataQueue.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+     * Pull data
+     * @param timeout
+     * @param unit
+     * @return
+     */
+    public StatData pull(long timeout, TimeUnit unit) {
+        try {
+            StatData message = queue.poll(timeout, unit);
+            return message;
+        } catch (InterruptedException ex) {
```
Comment (aloyszhang) [inline inlong-audit/audit-service/src/main/java/channel/DataQueue.java:62]: ditto here.

**Review by gosonzhang — APPROVED** [review]

**Review by aloyszhang — APPROVED** [review]

**Pull request merged**